src/SDL12_compat.c

```diff
@@ -8162,6 +8175,12 @@ DECLSPEC12 void SDLCALL
 SDL_GL_SwapBuffers(void)
 {
     if (VideoWindow20) {
+        if(OpenGLBuffersSwapTickInteval != 0.f) {
+            const Uint32 tickDelta = SDL20_GetTicks() - OpenGLBuffersLastSwapTicks;
+            if(tickDelta < OpenGLBuffersSwapTickInteval) {
+                SDL20_Delay( round(OpenGLBuffersSwapTickInteval) - tickDelta );
```

Comment:
The MSVC linking error is about `round` not being available (sdl12-compat is built without C runtime libraries).
Would casting `OpenGLBuffersSwapTickInterval` to an `int` work too?

Reply:
that works too, thanks!